tests/test_thunderbird_urls.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import pytest
+import requests
+
+
+@pytest.mark.skip_selenium
+@pytest.mark.nondestructive
+class TestThunerbirdURLs(object):
+
+    paths = [
+        '/thunderbird/all/',
+        '/thunderbird/releases/',
+    ]
+
+    def test_urls_ok(self, mozwebqa):
+        for path in self.paths:
```

Comment:
If you look at the other tests in this suite that check a number of urls, you'll see that we generally check each url in a loop and build a failure message that includes all failing urls, rather than doing an assert on each url. The problem with the latter approach, that you are taking here, is that the test will fail as soon as it finds a bad url, which means we won't be checking any of the other urls.